app_test_suite/steps/upgrade_test_runner.py

```diff
@@ -14,7 +14,7 @@
 from pytest_helm_charts.giantswarm_app_platform.entities import ConfiguredApp
 from pytest_helm_charts.giantswarm_app_platform.utils import delete_app
 from requests import RequestException
-from validators import url as validator_url
+# from validators import url as validator_url
```

Comment:
I got an error `ModuleNotFoundError: No module named 'validators'` unless I commented this out.

Reply:
Hmm, but `validators` are in Pipfile. I'll check it out later.